manifests/config.pp

```diff
@@ -158,11 +158,25 @@
           content => template("rsnapshot/${script}.sh.erb"),
           mode    => '0755',
         }
-        
+
       }
     }
 
-    $cronfile = "${cron_dir}/${host}"
+    # cron on Debian seems to ignore files that have dots in their name; replace
+    # them with underscores (issue #2)
+    if $::osfamily == 'Debian' {
+      $cron_name = regsubst("${host}", '\.', '_')
```

Comment:
If you happen to already have a patch, which fulfills all aforementioned requirements, feel free to issue a pull request yourself :) Seems to be quite some interest for this module to work on debian.
Thank you for your contribution!